tests/test_data/core/core__count_documentreference_month.txt

```diff
@@ -1,18 +1,18 @@
-(22, datetime.date(2018, 6, 1), 'ambulatory', None)
```

Comment:
Summary of changes here:
- Going through single entrypoint for counts switched datetime to str
- Reordered to accomodate template structure due to all the secondary/tertiary conditions